### PR TITLE
Feature/bus 795 three collums product boxes

### PR DESCRIPTION
**Pull request by carojansing**
Adds new CMS Block for three collums product boxes

### CODE REVIEW

File: components/cms/element/CmsElementProductBox.vue [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<script setup lang="ts">
+import type { CmsElementProductBox } from '@shopware-pwa/composables-next';
+import ProductCard from '~/components/product/ProductCard.vue';
```
Comment (aheartforspinach) [inline components/cms/element/CmsElementProductBox.vue:3]:
```suggestion
```

File: components/cms/element/CmsElementProductBox.vue [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+const props = defineProps<{
+    element: CmsElementProductBox;
+}>();
+
+const { product } = useProduct(props.element.data.product);
+</script>
+
+<template>
+        <ProductCard :product="product"></ProductCard>
```
Comment (aheartforspinach) [inline components/cms/element/CmsElementProductBox.vue:13]: The space seems a little bit too much? Did you run bun run lint?

**Review by aheartforspinach — APPROVED** [review]

**Review by mloeffle — APPROVED** [review]

**Review by m-borgmann — APPROVED** [review]

File: components/cms/element/CmsElementProductBox.vue [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+const props = defineProps<{
+    element: CmsElementProductBox;
+}>();
+
+const { product } = useProduct(props.element.data.product);
+</script>
+
+<template>
+    <ProductCard :product="product"></ProductCard>
```
Comment (m-borgmann) [inline components/cms/element/CmsElementProductBox.vue:12]:
```suggestion
    <ProductCard :product="product" />
```

**Comment by aheartforspinach** [comment]
@carojansing could you merge? you got all approvals last week

**Pull request merged**